targetplatform/JBossTools Oxygen Linux x86_64.launch

```diff
@@ -21,8 +21,8 @@
.runtime@default:default,javaewah@default:default,javassist@default:default,javax.activation@default:default,javax.annotation@default:default,javax.ejb@default:default,javax.el@default:default,javax.inject@default:default,javax.jws@default:default,javax.mail@default:default,javax.persistence@default:default,javax.servlet.jsp@default:default,javax.servlet@default:default,javax.transaction@default:false,javax.validation@default:default,javax.websocket-api@default:default,javax.ws.rs@default:default,javax.wsdl*1.5.1.v201012040544@default:default,javax.wsdl*1.6.2.v201012040545@default:default,javax.xml.bind@default:default,javax.xml.rpc@default:default,javax.xml.soap@default:default,javax.xml.stream@default:default,javax.xml.ws@default:default,javax.xml@default:default,jnr.x86asm@default:default,minimal-json@default:default,net.sourceforge.lpg.lpgjavaruntime@default:default,org.aopalliance@default:default,org.apache.ant@default:default,org.apache.aries.spifly.dynamic.bundle@default:default,org.apache.aries.util@default:default,org.apache.axis@default:default,org.apache.batik.css*1.7.0.v201011041433@default:default,org.apache.batik.css*1.8.0.v20170214-1941@default:default,org.apache.batik.dom@default:default,org.apache.batik.ext.awt@default:default,org.apache.batik.svggen@default:default,org.apache.batik.util*1.7.0.v201011041433@default:default,org.apache.batik.util*1.8.0.v20170214-1941@default:default,org.apache.batik.util.gui*1.7.0.v200903091627@default:default,org.apache.batik.util.gui*1.8.0.v20170214-1941@default:default,org.apache.batik.xml@default:default,org.apache.bcel@default:default,org.apache.commons.beanutils@default:default,org.apache.commons.codec*1.6.0.v201305230611@default:default,org.apache.commons.codec*1.9.0.v20170208-1614@default:default,org.apache.commons.collections@default:default,org.apache.commons.compress@default:default,org.apache.commons.discovery@default:default,org.apache.commons.httpclient@default:default,org.apache.commons.io@default:default,org.apache.commons.jxpath@default:default,org.apache.commons.lang3.source@default:default,org.apache.commons.lang3@default:default,org.apache.commons.lang@default:default,org.apache.commons.logging*1.0.4.v201101211617@default:default,org.apache.commons.logging*1.1.1.v201101211721@default:default,org.apache.commons.math@default:default,org.apache.commons.net@default:default,org.apache.commons.pool@default:default,org.apache.felix.gogo.command@default:default,org.apache.felix.gogo.runtime@default:default,org.apache.felix.gogo.shell@default:default,org.apache.felix.scr@default:default,org.apache.httpcomponents.httpclient@default:default,org.apache.httpcomponents.httpcore@default:default,org.apache.jasper.glassfish@default:default,org.apache.log4j@default:default,org.apache.lucene.analyzers-common@default:default,org.apache.lucene.analyzers-smartcn@default:default,org.apache.lucene.core*3.5.0.v20120725-1805@default:default,org.apache.lucene.core*6.1.0.v20170814-1820@default:default,org.
```

Comment:
just asking to double-check if this file has been changed on purpose

Reply:
will roll back